app/services/foreman/plugin/rbac_support.rb

```diff
@@ -7,6 +7,13 @@ def add_all_permissions_to_default_roles(all_permissions)
           add_all_permissions_to_role("Manager", all_permissions)
           add_all_permissions_to_role("Viewer", view_permissions)
         end
+      rescue ActiveRecord::StatementInvalid => e
+        if Foreman.in_rake? || Rails.env == 'test'
```

Comment:
Isn't this supposed to be `Rails.env != 'test'`? Because in test environment we want to rather error out (and definitely not want to flood logs).